dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -302,12 +302,15 @@ static void Edit_BackWord(HWND hwndEdit)
         return;
     }
 
-    for (; 0 < iStart; --iStart)
+    WORD wType1, wType2;
+    for (--iStart; 0 < iStart; --iStart)
```

Comment:
is `iStart` guaranteed to be smaller than `cchText` ?

Reply:
I think `iStart` is `<= cchText`.

Reply:
I added a check `(cchText < iStart)`. OK.